### ISSUE TITLE
Inverting source of truth for setup instructions

### ISSUE THREAD

**Issue by razzeee** [flatpak/flatpak.github.io#674]
Status quo:

This repo holds the data for setup instructions.

We sync it over to flathub.org with a script.

Proposal: 

Invert this flow and get the data from flathub.org

Benefit:

We want to start having setup instructions translated, see https://github.com/flathub-infra/website/issues/3118

**Comment by AsciiWolf** [comment]
Interesting idea, however I am not sure whether such auto-sync would be possible on Flatpak.org with the current deploy setup.

@barthalion What do you think? :)

**Comment by razzeee** [comment]
I'm not sure if I said auto sync, but just some script, that somebody runs from time to time, basically the same as we have on flathub right now.

https://github.com/flathub-infra/website/blob/main/frontend/setup-instructions.sh

**Comment by barthalion** [comment]
Yeah, we could be cloning this in the other direction, although I would rather not learn how to re-use translations with Jekyll. As long as file structure stays the same, I think it's alright.

**Comment by razzeee** [comment]
Yeah, my expectation is to only use english here for now.
